components/sidebar/add_channel_dropdown.tsx

```diff
@@ -74,14 +88,15 @@ class AddChannelDropdown extends React.PureComponent<Props, State> {
             <Menu.ItemAction
                 id={'browseDirectMessages'}
                 onClick={this.props.handleOpenDirectMessagesModal}
-                icon={<i className='icon-account-plus-outline'/>}
+                icon={<i className='icon-account-outline'/>}
```

Comment:
Is this an intended change?

Reply:
yes! it changed to give the invite people button  the plus-outline.

Reply:
Oh I see, we gave the new menu item the plus and removed it from the DM. Gotcha!